rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/parsing/RpgParserWithMuteRuntimeTest.kt

```diff
@@ -139,7 +139,6 @@ class RpgParserWithMuteRuntimeTest {
 
     @Test
     fun executingFIZZBUZZTEST() {
-        DummySystemInterface.executedAnnotationInternal.clear()
```

Comment:
This helped with fuzzy tests before. Is this not needed anymore?

Reply:
We don't use the DummySystemInterface here, but the SimpleSystemInterface, so I think that the cleaning is useless here, isn't it?